packages/pieces/community/interfaces/src/index.ts

```diff
@@ -0,0 +1,13 @@
+import { createPiece, PieceAuth } from '@activepieces/pieces-framework';
+import { interfaceTrigger } from './lib/triggers/interface-trigger';
+import { returnResponse } from './lib/actions/return-response';
+
+export const interfaces = createPiece({
+  displayName: 'Interfaces',
```

Comment:
I remember we discussed "Forms" vs "Interfaces".

"Interfaces" is a too generic word for a form 
"form" would give the feeling that it stores responses and normal form software.

I don't really have other suggestions but please try to think about it and let see if there is better word